OpenRA.Platforms.Default/OpenAlSoundEngine.cs

```diff
@@ -483,6 +491,107 @@ public virtual void Stop()
 		}
 	}
 
+	class OpenAlAsyncLoadSound : OpenAlSound
+	{
+		static readonly byte[] SilentData = new byte[2];
+		readonly CancellationTokenSource cts = new CancellationTokenSource();
+		readonly Task playTask;
+
+		public OpenAlAsyncLoadSound(uint source, bool looping, bool relative, WPos pos, float volume, int channels, int sampleBits, int sampleRate, Stream stream)
+			: base(source, looping, relative, pos, volume, sampleRate)
+		{
+			// Load a silent buffer into the source. Without this,
+			// attempting to change the state (i.e. play/pause) the source fails on some systems.
+			var silentSource = new OpenAlSoundSource(SilentData, channels, sampleBits, sampleRate);
```

Comment:
You missed the `AL10.alSourcei(source, AL10.AL_BUFFER, (int)silentSource.Buffer);` here!

Reply:
With this addition it now works correctly on my machine.